### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by mmsesay**
In this PR I did the following:

- Configure linters
- Create the homepage and about page
-  Add images to the pages
- Add dynamic data from javascript to the main program section

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  eslint:
+    name: ESLint
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:50]: Fix your linter issues on your Pull Request.

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+About second view                                 |  About footer view
+:------------------------------------------------:|:-----------------------------------------------:
+![](./public/images/screenshots/m5.png)           |  ![](./public/images/screenshots/m6.png)
+
+## Built With
+
+- HTML & CSS & JAVASCRIPT
+
+## Link to project online
```
Comment (carlos-ssh) [inline README.md:25]: **Provide feedback about the video presentation in the code review summary.**
1. A video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. A clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
4. A demo of the project, showing the pages in different screen sizes.
5. A highlight about a piece of the code the student finds interesting or is proud of making.

>

File: README.md [hunk 3]
```diff
@@ -1 +1,48 @@
...
+- [OnlineVersion](https://mmsesay.github.io/microverse-capstone-project-one/)
```
Comment (carlos-ssh) [inline README.md:26]: **Your Live Demo isn't working well.**
- All pages, links, and images work as expected.
_Kindly fix it._
Screen Shot 2021-09-07 at 5 43 47

File: about.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./public/css/style.css" />
```
Comment (carlos-ssh) [inline about.html]: - Kindly give style in the about page. 
Screen Shot 2021-09-07 at 5 49 04

File: public/js/index.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
+// index.js
```
Comment (carlos-ssh) [inline public/js/index.js:1]: [OPTIONAL]
Keep your project clean and professional.
- Remove commented code.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
```
Comment (carlos-ssh) [inline index.html:1]: Provide a Desktop Size:

1. That divides in a top dark bar and a bottom white bar.
2. The top short bar section with social media links, logout, and language links aligned to the right.
3. The taller bottom bar with a logo and a list of page links.
4. The top bar contains a logo to the left.
5. The top bar contains a list of page links aligned to the right.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mmsesay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (mmsesay) [reply .github/workflows/linters.yml:50]: I have resolved this issue. You can check it again.

File: about.html (hunk 4 above)
Comment (mmsesay) [reply about.html]: I have resolved this issue. You can check it again.

File: README.md (hunk 3 above)
Comment (mmsesay) [reply README.md:26]: I have made the required changes as suggested

File: index.html (hunk 6 above)
Comment (mmsesay) [reply index.html:1]: I have made the required changes as suggested

File: .github/workflows/linters.yml (hunk 1 above)
Comment (henatan99) [reply .github/workflows/linters.yml:50]: > Fix your linter issues on your Pull Request.

> I have resolved this issue. You can check it again.

This issue is resolved. Nice work! :clap:

File: README.md (hunk 2 above)
Comment (henatan99) [reply README.md:25]: This issue is nicely addressed. Good job! :clap: :+1:

File: README.md (hunk 3 above)
Comment (henatan99) [reply README.md:26]: > **Your Live Demo isn't working well.**
> 
> * All pages, links, and images work as expected.
>   _Kindly fix it._
>   Screen Shot 2021-09-07 at 5 43 47


- [x] This issue is not resolved yet. There are still overlapping components (in the navbar) for bigger screens (768 and beyond). Kindly fix them. :pray: 
image
- [x] There are also components that go outside of the view window (the logo) for the smaller screen(up to 768). Please, make changes to this one.
image

File: index.html (hunk 6 above)
Comment (henatan99) [reply index.html:1]: > Provide a Desktop Size:
> 
> 1. That divides in a top dark bar and a bottom white bar.
> 2. The top short bar section with social media links, logout, and language links aligned to the right.
> 3. The taller bottom bar with a logo and a list of page links.
> 4. The top bar contains a logo to the left.
> 5. The top bar contains a list of page links aligned to the right.



> I have made the required changes as suggested

This issue is addressed. Thank you for making the requested changes! :clap:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mmsesay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (rloterh) [reply README.md:26]: This issue is well resolved! ✔️

File: index.html (hunk 6 above)
Comment (rloterh) [reply index.html:1]: This issue is well resolved! ✔️

File: README.md [hunk 7]
```diff
@@ -1 +1,55 @@
...
+
+## Link to project recorded video
+- [OnlineVersion](https://www.loom.com/share/ea911f26628a4ed1919244f1a0dedd7f)
+
+## Authors
+
+👤 **Author1**
```
Comment (rloterh) [inline README.md:33]: - [x] OPTIONAL
Kindly replace the text **Author1** with your full name, so as to make your ReadMe more professional.

File: about.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    />
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-QFNS01QLGM"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
```
Comment (rloterh) [inline about.html:19]: - [x] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+                <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">News</a></li>
+                <li class="subHeaderListStyle campaign"><a href="#" class="no-underline secondary-text-color">Music Campaign</a></li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+        <i class="fas fa-bars cursor-pointer" id="openMenuIcon"></i>
+      </header>
+
+      <!-- Headline section -->
```
Comment (rloterh) [inline index.html:59]: - [x] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Muhammad, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. I must commend you, **your design is fantastic!** 🌟🌟🌟 ... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (mmsesay) [reply index.html:59]: okay, thanks for the suggestion. I will try to remove them.

File: about.html (hunk 8 above)
Comment (mmsesay) [reply about.html:19]: Thanks for this suggestion. I will try to use arrow function instead of normal functions

File: README.md (hunk 7 above)
Comment (mmsesay) [reply README.md:33]: Okay.

**Pull request merged**